docs/build.md

```diff
@@ -0,0 +1,19 @@
+# build
+
+This subcommand builds the Lenra app of the current directory.
+The app configuration are defined by a [configuration file](#configuration-file).
+
+```bash
+$ lenra build --help
+lenra-build 
+Build your app in release mode
+
+USAGE:
+    lenra build [OPTIONS]
+
+OPTIONS:
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+        --expose <EXPOSE>    Exposes services ports [default: app postgres mongo] [possible values:
```

Comment:
Does this mean that not setting the `--expose` parameter will result into the CLI assigning by default the `--expose app postgres mongo` ?

Reply:
No, what I wanted is that if the expose is not set, the expose is empty, but if you use the expose option without value, the app, mongo and postgres services are exposed.

This is not working yet, I'm working on it.

Reply:
I have just pushed the fix